op-node/chaincfg/chains.go

```diff
@@ -43,36 +43,44 @@ var Mainnet = rollup.Config{
 var Goerli = rollup.Config{
 	Genesis: rollup.Genesis{
 		L1: eth.BlockID{
```

Comment:
I think you edited the wrong network here.

Reply:
Oh yeah i totally did, good catch